p2p/net/swarm/swarm_addr_test.go

```diff
@@ -70,9 +70,11 @@ func TestFilterAddrs(t *testing.T) {
 		t.Fatal("should have failed to create swarm")
 	}
 
+	/* I'm not sure how i feel about trying to listen on bad addresses succeeding
```

Comment:
no, this one should definitely be tested, and it should succeed. the point is that those addresses may be deemed good or bad at runtime, and may be fed into the swarm. example, addr is in config, but doesnt work _right now_ with interfaces (say no ipv6 atm).